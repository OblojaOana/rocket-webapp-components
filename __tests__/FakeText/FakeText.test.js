import React from 'react';
import renderer from 'react-test-renderer';
import LoadingFakeText from '../../components/FakeText/LoadingFakeText';

describe('LoadingFakeText', () => {
    it('snapshot', () => {
        const component = renderer.create(<LoadingFakeText lines={10} />);

        const json = component.toJSON();
        expect(json).toMatchSnapshot();
    });
});