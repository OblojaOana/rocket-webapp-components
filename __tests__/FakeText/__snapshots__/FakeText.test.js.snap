// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingFakeText snapshot 1`] = `
<div
  className="makeStyles-wrapper-1"
>
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
  <div
    className="makeStyles-fake-2"
  />
</div>
`;
